src/bjointsp/parameters/templates/fw1chain.yaml

```diff
@@ -9,22 +9,31 @@ vnfs:
     inputs_bwd: 0
     outputs_fwd: 1
     outputs_bwd: 0
-    cpu: [0]
```

Comment:
Please avoid changes that are unrelated to VNF processing delays. Setting CPU requirements for the user "vnf" dosn't really make sense

Reply:
I copied this back from fw2chain.yaml after I made some changes and was unsure of what it was like before. In that file and fw3chaib.yaml the user vnf has both cpu and mem requirements.

Reply:
No problem. You can always easily check the file diffs in the pull request (here). It'll show you exactly what you changed